tests/backends/test_sparqlwrapper_graphdb_fuseki.py

```diff
@@ -56,7 +56,33 @@ def fuseki_available():
         time.sleep(interval)
 
 
-def populate_and_search(ts):
+def get_triplestore(tsname: str) -> "Triplestore":
+    """Help function that returns a new triplestore object."""
+    from tripper import Triplestore
+
+    if tsname == "GraphDB":
+        ts = Triplestore(
+            backend="sparqlwrapper",
+            base_iri="http://localhost:7200/repositories/test_repo",
+            update_iri=(
+                "http://localhost:7200/repositories/test_repo/statements"
+            ),
+        )
+    elif tsname == "Fuseki":
+        ts = Triplestore(
+            backend="sparqlwrapper",
+            base_iri=f"{FUSEKI_CHECK_URL}/test_repo",
+            update_iri=f"{FUSEKI_CHECK_URL}/test_repo/update",
+            username="admin",
+            password="admin0",
+        )
+    else:
+        raise ValueError(f"Unsupported triplestore name: {tsname}")
+
+    return ts
```

Comment:
I disagree with moving this out of the actual test for each triplestore. There is reduction in  duplication of code here, and the test as it is, is so small already that moving things out makes it less readable.

Reply:
The point with this is to make it easy to run the test without pytest for debugging. 

Now I can do:

```python
if True:
    tsname = "Tripper"
#def populate_and_search(tsname):
    ...
```

Reply:
You could run if True on test_fuseki and test_graphdb directly before.

Reply:
That will not allow me to debug. I want to be able to investigate variables within `populate_and_search()` with the interpreter.